Add installation notes to Readme.md (#13)
s-hamann/desec-dns: README.md

                        the request. This is the default behaviour.
```

Installation

> =====

Reviewer: ```suggestion
============
```
I just like my markdown pretty :wink:

Author: Done

Currently, the only way to install / use desec-dns is from source.

> You can simply run `python desec.py` as-is, as long as `requests` is installed for your
> python interpreter. Note that some features requre additional dependencies (see above).

Reviewer: How about this? Deduplicate some information.
```suggestion
You can simply run `python desec.py` as-is, as long as the dependencies (see
[above](#requirements)) are installed for your python interpreter.
```

Author: Done

The package can be installed in a (virtual) environment using 

> `pip install -e ./path/to/desc-dns`. That should also make `desec` available as a

Reviewer: There's also the the option of installing the package directly from GitHub without cloning explicitly:
```
pip install git+https://github.com/s-hamann/desec-dns
``` 
Should this be mentioned, too? It's not terribly important to me, as I hope to publish to PyPI soonish.

Author: Done
